dev-docs/modules/prebidServer.md

```diff
@@ -96,6 +96,7 @@ There are many configuration options for s2sConfig:
 | `defaultTtl` | Optional | Integer | Configures the default TTL in the Prebid Server adapter to use when Prebid Server does not return a bid TTL - 60 if not set |
 | `adapterOptions` | Optional | Object | Arguments will be added to resulting OpenRTB payload to Prebid Server in every impression object at request.imp[].ext.BIDDER. See the example above. |
 | `extPrebid` | Optional | Object | Arguments will be added to resulting OpenRTB payload to Prebid Server in request.ext.prebid. See the examples below. |
+| `customHeader` | Optional | Object | These custom headers will be included in the XHR call to the bidder's endpoint. This will allow you to send data specific to your use case. The format consists of an object where the keys represent the header names and the values correspond to the respective header values. |
```

Comment:
Thanks for the docs @sanved77 , but I think it would be useful to describe what's in the object with an example. I think an example value would be `{ "Header1": "Value1", "Header2": "Value2"}`, right?

Reply:
Makes sense to me. Will add that.

Reply:
Added.